kernel-headers/linux/mlx5/mlx5_ifc.h

```diff
@@ -6214,10 +6214,10 @@ struct mlx5_ifc_destroy_qp_in_bits {
 	u8         reserved_at_40[0x8];
 	u8         qpn[0x18];
 
-	u8         reserved_at_60[0x1];
-	u8         cmd_on_behalf[0x1];
-	u8         reserved_at_42[0xe];
-	u8         function_id[0x10];
+	u8        reserved_at_60[0x1];
```

Comment:
It's another indentation problem - lines should start with tab character - like other do.